docs/vickywane-ambianic-stripe-swagger-docs-0.1-resolved.yaml

```diff
@@ -0,0 +1,51 @@
+---
+openapi: 3.0.1
+info:
+  title: Ambianic Serverless functions API Definition
+  description: An API Definition for Ambianic Serverless functions created using Swagger
+  version: "0.1"
+servers:
+- url: https://ambianic-functions.netlify.app
+- url: https://14fecb7f-38f9-43ce-ae90-9381932906cf.mock.pstmn.io
+paths:
+  /create-stripe-subscriber:
+    post:
+      description: Serverless function for creating a customer on stripe and subscribing them to an item. 
+      responses:
+        "200":
+          description: Returns stripe customer response.
+          content:
+            text/html; charset=utf-8:
+              schema:
+                type: string
+              examples: {}
+      servers:
+      - url: https://14fecb7f-38f9-43ce-ae90-9381932906cf.mock.pstmn.io
+    servers:
+    - url: https://14fecb7f-38f9-43ce-ae90-9381932906cf.mock.pstmn.io
+  /.netlify/functions/getCustomers:
+    get:
+      description: Serverless function to fetch all customers from a stripe service account.
+      responses:
+        "200":
+          description: Returns an array containing stripe's customer object data.
+      servers:
+      - url: https://ambianic-functions.netlify.app
+    servers:
+    - url: https://ambianic-functions.netlify.app
+  /cancel-subscription:
```

Comment:
@vickywane this should probably be `/subscription` or `/subscriptions` with a delete operator matched to an HTTP DELETE method instead of GET.

Reply:
This has now been renamed to subscribtions.